src/main/java/mate/jdbc/dao/impl/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,131 @@
+package mate.jdbc.dao.impl;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.lib.DataProcessingException;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+    private final ConnectionUtil connectionUtil = new ConnectionUtil();
+
+    private Manufacturer parseManufacturer(ResultSet resultSet) {
+        try {
+            return new Manufacturer(
+                    resultSet.getObject("id", Long.class),
+                    resultSet.getString("name"),
+                    resultSet.getString("country")
+            );
+        } catch (SQLException e) {
+            throw new DataProcessingException(
+                    "cant get manufacturer from resultSet" + resultSet, e
+            );
+        }
+    }
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String sqlCode = "INSERT INTO taxi_db.manufacturers(name, country) "
+                + "VALUES(?, ?);";
```

Comment:
```suggestion
        String createRequest = "INSERT INTO manufacturers(name, country) "
                + "VALUES(?, ?);";
```

here you can not specify the name of the base